blockchain/scriptval.go

```diff
@@ -101,10 +108,11 @@ out:
 			if err := vm.Execute(); err != nil {
 				str := fmt.Sprintf("failed to validate input "+
 					"%s:%d which references output %s:%d - "+
-					"%v (input script bytes %x, prev output "+
-					"script bytes %x)", txVI.tx.Hash(),
-					txVI.txInIndex, originTxHash,
-					originTxIndex, err, sigScript, pkScript)
+					"%v (input witness %v, input script "+
```

Comment:
Switched to use `%x` instead which will recursively display all items using a hex encoding.